.github/ISSUE_TEMPLATE/add-new-component.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Add New Component
+about: Use this template when adding a new component to the UI-Kit
+
+---
+
+#### Summary
+
+#### Design
+
+<!-- provide a link to the original design -->
+
+[]()
+
+#### Jira Ticket
+
+[]()
+
+#### UI-Kit Component Checklist
+
+##### Technical
+
+- Documentation
+  - [ ] Has `README.md` file
+  - [ ] Explains use-cases
+  - [ ] Explains props
+  - [ ] Added to Storybook
+- Props
+  - [ ] Uses prop names consistent with existing components
+  - [ ] Does not use `true` for default-props
+  - [ ] Does not accept `classNames` prop
+  - [ ] Has default props for labels (where applicable)
+- Design
+
+  - [ ] Does not use hard-coded values
+  - [ ] Uses design-tokens (instead of e.g. color variables)
+
+- Testing
+  - [ ] Supports `data` attributes
+
+##### Design
+
+- [ ] Is added to Sketch library
```

Comment:
Does it make sense to expose this to the outside world?

Reply:
Not really. We will eventually generate the Sketch-library from here (we had a PoC for this already). Then we can remove it completely. I'm also fine with removing it now.